protos/icmp/icmp.go

```diff
@@ -0,0 +1,329 @@
+package icmp
+
+import (
+	"net"
+	"time"
+
+	"github.com/elastic/libbeat/common"
+	"github.com/elastic/libbeat/logp"
+	"github.com/elastic/libbeat/publisher"
+
+	"github.com/elastic/packetbeat/config"
+	"github.com/elastic/packetbeat/protos"
+
+	"github.com/tsg/gopacket/layers"
+	// TODO: better use this? (also see commit #16691f7)
+	// "github.com/google/gopacket/layers"
```

Comment:
Yes I know about additions. I also have pull request open (https://github.com/google/gopacket/pull/134) but nobody is looking at it.